### DISCUSSION TITLE
MacOS M1 Mac Mini --Volume ????? doesnt work (`stat /Users/Shared/jdownloader2: no such file or directory`)

### DISCUSSION THREAD

**Discussion by cracyfloyd** [containerd/nerdctl#1075] [Q&A] [answered]
I use Rancher Desktop without Lima ! 
Follow Error occours !!
failed to create shim: OCI runtime create failed: container_linux.go:380: starting container process caused: process_linux.go:545: container init caused: rootfs_linux.go:76: mounting "/Users/Shared/jdownloader2" to rootfs at "/jDownloader2" caused: stat /Users/Shared/jdownloader2: no such file or directory: unknown 

My example i try:
nerdctl run --name jDownloader2 -d \
> -p 8080:8080 \
> --env 'CUSTOM_RES_W=1024' \
> --env 'CUSTOM_RES_H=768' \
> --volume /Users/Shared/jdownloader2:/jDownloader2 \
> --volume /Users/Shared/jdownloader2/downloads:/mnt/jDownloader \
> ich777/jdownloader2:arm64

Please help !!!

**Comment by AkihiroSuda** [comment, marked as answer]
> I use Rancher Desktop without Lima !

If you are using Rancher Desktop on macOS, you are already using Lima.

> mounting "/Users/Shared/jdownloader2" to rootfs at "/jDownloader2" caused: stat /Users/Shared/jdownloader2: no such file or directory

This is because Rancher Desktop does not mount `/Users/Shared` into its internal Lima VM.
You can submit a future request to https://github.com/rancher-sandbox/rancher-desktop/issues .

**Reply by cracyfloyd** [reply]
Thnx. And which folder i can use ??
All is new for me because i am coming from my linux raspbian system an this my first trys to implement it in mac os arm...

**Reply by AkihiroSuda** [reply]
`/Users/<User Name>` and`/tmp/rancher-desktop`, as of Rancher Desktop v1.3.0.
